lib/cluster/election.c

```diff
@@ -246,6 +308,21 @@ election_vote(election_t *e)
     return;
 }
 
+/*!
+ * \brief Check whether local node has won an election
+ *
+ * If all known peers have sent no-vote messages, stop the election timer, set
+ * the election state to won, and call any registered win callback.
+ *
+ * \param[in] e      Election object
+ *
+ * \return TRUE if local node has won, FALSE otherwise
+ * \note If all known peers have sent no-vote messages, but the election owner
+ *       does not call this function, the election will not be won (and the
+ *       callback will not be called) until the election times out.
+ * \note This should be called when election_count_vote() returns
+ *       election_in_progress.
```

Comment:
just a cosmetic thing towards nice generated documentation,
`\c election_in_progress` might serve better (haven't checked
but don't expect it would be picked and teletype-formatted automatically

Reply:
will do